src/commands/ai/extract-structured.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+const BoxCommand = require('../../box-command');
+const {Flags} = require('@oclif/core');
+const utils = require('../../util');
+
+class AiExtractStructuredCommand extends BoxCommand {
+	async run() {
+		const {flags} = await this.parse(AiExtractStructuredCommand);
+		let options = {};
+
+		if (flags.items) {
+			options.items = flags.items;
+		}
+
+		if (flags.fields) {
+			options.fields = flags.fields;
+			delete this.flags.fields;
+		} else if (flags['metadata-template']) {
+			options.metadataTemplate = flags['metadata-template'];
+		} else {
+			throw new Error('Either fields or metadata_template must be provided');
```

Comment:
Should also check if both fields and metadata template provided at same time, if yes throw an error